kano/webapp.py

```diff
@@ -94,6 +94,9 @@ def run(self):
         view.connect('close-web-view', self._close)
         view.connect('onload-event', self._onload)
 
+        language = self._get_browser_lang()
+        view.execute_script("window.navigator.userLanguage = '%s'" % language)
```

Comment:
Thats strange, it worked for me when my system is localised to `as_AR.UTF-8`. Can you show me the output of `locale` in the shell for you?